doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -1,84 +1,92 @@
-==========================
-BlueStore Config Reference
-==========================
+==================================
+ BlueStore Configuration Reference 
+==================================
 
 Devices
 =======
 
-BlueStore manages either one, two, or (in certain cases) three storage
-devices.
+BlueStore manages either one, two, or in certain cases three storage devices.
 
-In the simplest case, BlueStore consumes a single (primary) storage device.
-The storage device is normally used as a whole, occupying the full device that
-is managed directly by BlueStore. This *primary device* is normally identified
-by a ``block`` symlink in the data directory.
+In the simplest case, BlueStore consumes a single storage device. The device is
+known as the *primary device*. Any device used by BlueStore will be used
+entirely by BlueStore and nothing other than BlueStore. The primary device is
+identified by a ``block`` symlink in the data directory.
 
-The data directory is a ``tmpfs`` mount which gets populated (at boot time, or
-when ``ceph-volume`` activates it) with all the common OSD files that hold
-information about the OSD, like: its identifier, which cluster it belongs to,
-and its private keyring.
+The data directory is a ``tmpfs`` mount. When this data directory is booted or
+activated by ``ceph-volume``, it is populated with all of the common OSD files
+that hold information about the OSD: for example, the OSD's identifier, the
+name of the cluster that the OSD belongs to, and the OSD's private keyring.
 
-It is also possible to deploy BlueStore across one or two additional devices:
+In more complicated cases, BlueStore is deployed across one or two additional
+devices:
 
-* A *write-ahead log (WAL) device* (identified as ``block.wal`` in the data directory) can be
-  used for BlueStore's internal journal or write-ahead log. It is only useful
-  to use a WAL device if the device is faster than the primary device (e.g.,
-  when it is on an SSD and the primary device is an HDD).
+* A *write-ahead log (WAL) device* (identified as ``block.wal`` in the data
+  directory) can be used for BlueStore's internal journal or write-ahead log.
```

Comment:
can be used to separate out BlueStore's internal

What I'm trying to do here is to reinforce the idea that WAL and DB aren't optional things to add to an OSD, but rather things that by default are colocated with the primary data but may be split out.

Reply:
Accepted.